src/loadext.c

```diff
@@ -554,19 +554,23 @@ static int sqlite3LoadExtension(
   char *zAltEntry = 0;
   void **aHandle;
   u64 nMsg = strlen(zFile);
+  #ifndef _FREEBSD_KERNEL
```

Comment:
You do not need these #ifdef guards if you configure the build with SQLITE_OMIT_LOAD_EXTENSION, which you should for now as you are not going to be using extensions in the kernel version of SQLite.

Reply:
Done.